antora-docs/modules/ROOT/pages/index.adoc

```diff
@@ -1,4 +1,10 @@
 
+////
+This content is automatically generated from a template, see
+https://github.com/hacbs-contract/ec-policies/tree/main/docsrc
+Do not edit it manually.
+////
+
 = HACBS Enterprise Contract Policies
```

Comment:
I think this will disable the "Edit this page" links, see [default-ui docs](https://docs.antora.org/antora-ui-default/templates/#template-variables)
```suggest
:page-origin-private: true
```

Reply:
Nice. I think there's some value of having a link to the source repo, so maybe we can let it stay as it is for now.